shadowrunners/margrond.md

```diff
@@ -0,0 +1,136 @@
+#Margrond 
+
+##Personal Data
+
+_Metatype:_ Troll
+
+* Thermographic Vision
+* +1 Reach
+* +1 Dermal Armor
+* +100% Lifestyle Costs
+
+[Character Creation Calculations](https://www.evernote.com/l/AMz2UDswvZJJ4rquVkiTklhYpYHjfaNKsl0)
+
+| Ethnicity | Age | Sex | Height | Weight |
+| --- | --- | --- | --- | --- |
+| - | - | - | - | - |
+
+| Street Cred | Notoriety | Public Awareness | 
+| --- | --- | --- |
+| n/a | n/a | n/a |
+
+| Karma | Total Karma | Misc | 
+| --- | --- | --- |
+| 0 | 47 | n/a |
+
+##Attributes
+| BOD | AGI | REA | STR | WIL | LOG | INT | CHA |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| 6 / 10 | 5 / 5 | 5 / 6 | 6 / 10 | 3 / 6 | 2 / 5 | 2 / 5 | 1 / 4 |
+
+| ESS | Edge | Initiative | Physical Limit | Mental Limit | Social Limit |
+| --- | --- | --- | --- | --- | --- |
+| 0.1 | 1 | 1d6 + 7 | 8 | 3 | 2 |
+
+## Condition Monitor
+__Physical:__ 11
+
+__Stun:__ 10
+
+__Overflow:__ 6
+
+##Augmentations
+| Augmentation | Rating/Attributes | Notes | Essence |
+| --- | --- | --- | --- |
+| Bone Lacing | BOD +2, Armor +2, Unarmed Damage (STR+2)P | - | 1.0 |
+| Cyberwear Commlink (Transys Avalon) | 6 | - | 0.2 |
+| Cybereyes | 3 | Flare Compensation, Low-light Vision, Vision Magnification, Vision Enhancement (1), Smartlink [Remaining Capacity 3] | 1.0 |  
+| Cyberears | 3 | Audio Enhancement (1), Damper [Remaining Capacity 10] | 0.7 |
+| Synthetic Arm (Left) | AGI 7, STR 8, Armor 2 | AGI Enhancement +2, STR Enhancement +2, Armor Enhancement +2, (+2 AGI, +3 STR Customization), -8 Concealable [Remaining Capacity 2] | 1.0 |
+| Synthetic Arm (Right) | AGI 7, STR 8, Armor 2 | AGI Enhancement +2, STR Enhancement +2, Armor Enhancement +2, (+2 AGI, +3 STR Customization), -8 Concealable [Remaining Capacity 2] | 1.0 |
+| Synthetic Leg (Left) | - | Cyber Holster, Large Smuggling Compartment, -8 Concealable | 1.0 |
+
+##Melee Weapons
+| Weapon | Accuracy | Reach | Damage | AP |
+| --- | --- | --- | --- | --- |
+| Forearm Snap-blades | 4 | - (1) | (STR+2)P | -2 |
+| Katana | 7 | 1 (2) | (STR+3)P | -1 |
+| Combat Axe | 4 | 2 (3) | (STR+5)P | -4 |
+
+##Ranged Weapons
+
+| Weapon | Damage | Accuracy | AP | Mode | RC | Ammo | Notes |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| FN HAR | 10P | 5(7) | -2 | SA / BF / FA | 2 | 35 (c) | Smartlink [integral], Periscope [Top], Gas-vent 2 [Barrel], Gyro Mount [Underbarrel]
+| Ares Predator V | 8P | 5(7) | -1 | SA | - | 15 (c) | Smartlink [integral], Periscope [Top], Gas-vent 3 [Barrel]|
```

Comment:
Here's my understanding of the rules, and why I tossed the same gas-vent on there when I spec'd out a similar pistol:

Methinks semi-auto pistols are going to build up recoil mainly when using the "semi-auto burst" complex action (p. 167). Compared to firing a normal semi-auto shot: it fires three bullets instead of three, it takes a complex action instead of simple, and it gives a sweet -2 defense penalty to whatever you're shooting. 

So load up on recoil compensation and burst away!